content/news/playtest-20150719.md

```diff
@@ -0,0 +1,46 @@
+We are pleased to announce the second playtest on the road to our next stable release.  With this playtest, we have entered the feature freeze phase, so it already is pretty close to what the release will eventually look like. There are probably a number of bugs still lurking in the code, so it needs testing, testing and some more testing!
```

Comment:
", so it already is pretty close to what the release will eventually look like." is really hard to parse.  Could we streamline this a bit?